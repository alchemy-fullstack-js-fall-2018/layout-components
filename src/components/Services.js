import Component from './Component';
import styles from './Services.css';

export default class Services extends Component {
  render() {
    const servicesSection = `
    <div id="services">
    <h3>Services</h3>
    <p>${this.props.service}</p> </div>`;

    return `<section class="${styles.services}">
      ${servicesSection}
    </section>`;
  }
}
