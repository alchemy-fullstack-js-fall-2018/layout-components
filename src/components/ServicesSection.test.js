import Services from "./Services";

describe('Services component', () => {
  it("should be a project section", () => {
    const service = new Services({
      service: ['Design', 'Programming', 'Freelancing']
    });
    expect(service.render()).toMatchSnapshot();
  });
});
