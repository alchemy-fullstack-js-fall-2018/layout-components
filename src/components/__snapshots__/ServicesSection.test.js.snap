// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Services component should be a project section 1`] = `
"<section class=\\"services\\">
      
    <div id=\\"services\\">
    <h3>Services</h3>
    <p>Design,Programming,Freelancing</p> </div>
    </section>"
`;
